erica_app/erica/request_processing/erica_input.py

```diff
@@ -89,48 +89,116 @@ class FormDataEst(BaseModel):
     stmind_gem_haushalt_count: Optional[int]
     stmind_gem_haushalt_entries: Optional[List[str]]
 
+    @root_validator(pre=True)
+    def disability_merkzeichen_and_pflegegrad_flags_must_be_boolean(cls, values):
+        merkzeichen_names = [
+            'person_a_has_pflegegrad',
```

Comment:
I just noticed: We shouldn't forget to set the pflegegrad as a bool field in the elster_client as it is a YesNoField, thus string by default. But more a future comment than for this PR.